packages/vscode-extension/src/debug/depsChecker/doctorConstant.ts

```diff
@@ -19,6 +19,8 @@ export const doctorConstant = {
   NodeNotFound: `Cannot find Node.js. ${InstallNode}`,
   NodeNotSupported: `Node.js (@CurrentVersion) is not in the supported version list (@SupportedVersions). ${InstallNode}`,
   NodeSuccess: `Supported Node.js version (@Version) is installed`,
+  BypassNode12: `To continue to local debug using Node.js v12, go to Visual Studio Code Settings, under Teams Toolkit, Prerequisite Check, uncheck "Ensure Node.js is installed.(node)".`,
+  BypassNode12AndFunction: `To continue to local debug using Node.js v12, go to Visual Studio Code Settings, under Teams Toolkit, Prerequisite Check, uncheck "Ensure Node.js is installed. (node)" and "Ensure Azure Functions Core Tools is installed (funcCoreTools)". Also make sure you install the Azure Functions Core Tools v3. https://github.com/Azure/azure-functions-core-tools`,
```

Comment:
Maybe I make it wrong in my last comment, but we have 3 styles here
Ensure Node.js is installed.(node) - with period without space
Ensure Node.js is installed. (node) - with period with space
Ensure Azure Functions Core Tools is installed (funcCoreTools) - without period and with space
We need to make it consistent - consistent to what we show in the settings.

Reply:
Has been unified.